packages/beacon-node/src/network/reqresp/handlers/dataColumnSidecarsByRange.ts

```diff
@@ -72,61 +79,6 @@ export async function* onDataColumnSidecarsByRange(
   }
 }
 
-export function* iterateDataColumnBytesFromWrapper(
-  chain: IBeaconChain,
-  dataColumnSidecarsBytesWrapped: Uint8Array,
-  blockSlot: Slot,
-  columns: ColumnIndex[]
-): Iterable<ResponseOutgoing> {
-  const retrivedColumnsLen = ssz.Uint8.deserialize(
-    dataColumnSidecarsBytesWrapped.slice(NUM_COLUMNS_IN_WRAPPER_INDEX, COLUMN_SIZE_IN_WRAPPER_INDEX)
-  );
-  const retrievedColumnsSizeBytes = dataColumnSidecarsBytesWrapped.slice(
-    COLUMN_SIZE_IN_WRAPPER_INDEX,
-    CUSTODY_COLUMNS_IN_IN_WRAPPER_INDEX
-  );
-  const columnsSize = ssz.UintNum64.deserialize(retrievedColumnsSizeBytes);
-  const dataColumnsIndex = dataColumnSidecarsBytesWrapped.slice(
-    CUSTODY_COLUMNS_IN_IN_WRAPPER_INDEX,
-    CUSTODY_COLUMNS_IN_IN_WRAPPER_INDEX + NUMBER_OF_COLUMNS
-  );
-  const allDataColumnSidecarsBytes = dataColumnSidecarsBytesWrapped.slice(
-    DATA_COLUMN_SIDECARS_IN_WRAPPER_INDEX + 4 * retrivedColumnsLen
-  );
-
-  const columnsLen = allDataColumnSidecarsBytes.length / columnsSize;
-
-  // no columns possibly no blobs
-  if (columnsLen === 0) {
-    return;
-  }
```

Comment:
@nazarhussain this case is what I mean, we previously skipped if block had no blobs while now we error